boards/xtensa/m5stack_atoms3/m5stack_atoms3_defconfig

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_BOARD_M5STACK_ATOMS3=y
+CONFIG_SOC_SERIES_ESP32S3=y
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_ESP_HEAP_MEM_POOL_REGION_1_SIZE=0
+
+CONFIG_GPIO=y
+CONFIG_REGULATOR=y # for LCD backlight
+
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+
+# for debugging
+CONFIG_SHELL=y
```

Comment:
I suggest removing this as default config. This will allow `test/lib/heap` to pass. User can manually enable this if necessary. Sounds good?

Reply:
Sure thing! Definitely the right thing to do. Pretty sure I copy pasted from another esp32 board fwiw, including the "for debugging" comment.